avocado/utils/memory.py

```diff
@@ -41,6 +108,20 @@ def read_from_meminfo(key):
     return int(re.search(r'\d+', meminfo).group(0))
 
 
+def swaptotal():
+    """
+    Read ``SwapTotal`` from meminfo.
+    """
+    return read_from_meminfo('SwapTotal')
```

Comment:
I have very few appreciation for functions that don't really add value to the library. I mean, what's the value of calling `swaptotal()` instead of calling `read_from_meminfo('SwapTotal')`?